cns/restserver/ipam.go

```diff
@@ -72,13 +81,17 @@
 		}
 
 		err = service.Listener.Encode(w, &resp)
+		logger.Printf("Service name is %s, operation name is %s, IP config request in CNS is %s, reserveResp is %v, return code is %d, return code to string is %s, err is %v",
```

Comment:
Similar comment here, no need to add the comment here. we should improve logger.Response